parkhangs-webapp/src/features/events/eventsSlice.js

```diff
@@ -117,20 +117,24 @@ const eventSlice = createSlice({
 
             state.updatingEvent = false
             state.error = null
-        },
+        }
+        ,
+
+        updateEventByIdFailure(state, action) {
 
-        updateEventByIdFailure (state, action) {
             state.updatingEvent = false
             state.error = action.payload
-        },
+        }
+        ,
 
-        deleteEventStart (state, action) {
+        deleteEventStart(state, action) {
             state.deletingEvent = true
             state.error = null
-        },
+        }
+        ,
```

Comment:
please put these commas back and also all of the other weird spacing stuff in this file